### ISSUE TITLE
PiP not working properly on Wayland

### ISSUE THREAD

**Issue by matheucomth** [telegramdesktop/tdesktop#25800]
### Steps to reproduce

1. open a video
2. picture-in-picture mode

### Expected behaviour

Video is expected to always overlay apps

### Actual behaviour

It happens that the video does not overlap the applications, the opposite happens, the applications end up overlapping the video
![Captura de tela de 2023-01-29 14-34-00](https://user-images.githubusercontent.com/33396386/215345432-11d1835b-4f5f-4af2-9474-2c7c4850f73a.png)
![Captura de tela de 2023-01-29 14-34-11](https://user-images.githubusercontent.com/33396386/215345438-25827eff-4f50-4c5f-84c4-6d2d955f76de.png)

### Operating system

Fedora 37

### Version of Telegram Desktop

4.5.8

### Installation source

Flatpak

**Comment by ilya-fedin** [comment]
I don't think this is fixable... It's a restriction of the Wayland protocol, you have to use some system-wide hotkey to make the window always-on-top.

**Comment by fishman** [comment]
The PiP window should at least have a distinguishible name. In sway using PiP just crashes the whole thing. But there is no way to set any attributes since the PiP window is just called TelegramDesktop.
